src/externalized/json/Json.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include "RustInterface.h"
+
+#include <string_theory/string>
+#include <vector>
+
+class JsonObject;
+
+class JsonValue {
+	public:
+		// Takes ownership of RJsonValue
+		JsonValue(RJsonValue* value) {
+			m_value.reset(value);
+		}
+		JsonValue(int value) {
+			m_value.reset(RJsonValue_fromInt(value));
```

Comment:
I would prefer it if the constructors used the member initialization list to set m_value. The optimizer may be able to handle this but it's better not to rely on it too much.